app/javascript/hyacinth_v1/components/digital_objects/rights/rights_form/AssetRightsForm.js

```diff
@@ -29,7 +29,11 @@ function AssetRightsForm(props) {
 
   const [rights, setRights] = useHash(merge({}, defaultAssetRights, initialRights));
 
-  const [updateRights, { error: updateError }] = useMutation(updateRightsMutation);
+  const [updateRights, { data: updateData }] = useMutation(updateRightsMutation);
```

Comment:
Same note here about wanting to include `error: updateError` and still show it in with `GraphQLErrors` component (which only renders if updateErrors are actually present).